drupal/sites/default/boinc/modules/boincwork/includes/boincwork.helpers.inc

```diff
@@ -1640,15 +1640,15 @@ function boincwork_tasktable($category = 0, $queryid = 1, $tselect = NULL, $app_
     ),
     array(
       'data' => bts('Run time', array(), NULL, 'boinc:task-table'),
-      'field'=>'elapsed_time',
+      'field'=> 'elapsed_time',
```

Comment:
Please use a consistent style for blanks around `=>`. In other occasions within this file there is a blank before and after the operator. Applies also to lines 1647 and 1651 below.